### PR TITLE
Start a typescript project

### PR DESCRIPTION
**Pull request by rachidlamouri**
## Description

Add package file, package lock,  tsconfig, sample entry file, and sample npm script.

## References

- [tsconfig](https://www.typescriptlang.org/tsconfig) reference
- [ts-node](https://www.npmjs.com/package/ts-node) typescript execution engine

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('You helplessly stare into the void as you try to gain your bearings. Your foot catches on what you believe to be a loose flagstone. You fall and break your neck. You are dead.');
```
Comment (jshemas) [inline index.ts:1]: Is this from https://github.com/jshemas/textAdventure/blob/gh-pages/adventure.json ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
+    "klicker-knight": "ts-node index.ts"
+  },
+  "devDependencies": {
+    "@tsconfig/node14": "^1.0.1",
```
Comment (jshemas) [inline package.json:6]: Should we use node16?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,11 @@
 # Klicker Knight
+
 The spearhead thrust into the untapped market of npm cli games
+
+## Local Development
+
+```bash
+npm install
```
Comment (jshemas) [inline README.md:8]: Should this be `npm ci`?

Comment (rachidlamouri) [reply README.md:8]: Nah. I think it's fine for the dependencies to update as we develop the project. If you run into issues you can always npm ci. Plus the CI script will catch dependency issues.

File: package.json (hunk 2 above)
Comment (rachidlamouri) [reply package.json:6]: Should probably just set it to 18

Comment (jshemas) [reply package.json:6]: I'm fine with 14. Can you set the `engines` property?

Comment (rachidlamouri) [reply package.json]: Updated everything to 16

**Review by jshemas — APPROVED** [review]

**Pull request merged**